Assets/Scripts/Combat/TargetIndicator.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TargetIndicator : MonoBehaviour
+{
+    private MouseSelect MouseSelector;
+    private const float IndicatorScaler = 1.25f;
```

Comment:
IndicatorScale, instead of IndicaterScaler. Scaler is a single-value quantity, and every `float` value is implicitly a scaler.

Reply:
Man now you gonna make me change all my `scalers` to `scale`?